learning_journal/routes.py

```diff
@@ -8,4 +8,4 @@ def includeme(config):
     config.add_route("home", "/")
     config.add_route("edit", "/edit")
     config.add_route("form", "/form")
```

Comment:
Is this line for creating new entries? The route name (and hell, the route itself) should illustrate that.